_pages/configuring-your-sp.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Configuring your Service Provider
+---
+
+# Configuring your Service Provider
+
+Once you have a SAML implementation, you'll need to configure your SP to work with login.gov. View our [SAML metadata](https://github.com/18F/identity-idp/wiki/SAML-Metadata) for all of the pertinent information. For an example, see our [demo SP application's configuration](https://github.com/18F/identity-sp-rails/blob/master/config/initializers/omniauth.rb). All authentication and logout requests must be signed. We require RSA SHA-256 signatures embedded with the authentication and logout requests. Here's some of the key information you'll need to configure on your side:
```

Comment:
I don't really like this section at all. Feels like there's much more we can do to help agencies understand the configuration process. Thing is, like you mention, it's _very_ language / stack dependent, so most just follow docs elsewhere (for example, [Spring Security SAML](http://projects.spring.io/spring-security-saml/)) and just need our IdP details.

Open to ideas on this one!